modules/ocf_mail/templates/site_vhost/mysql-alias-map.erb

```diff
@@ -0,0 +1,8 @@
+hosts = mysql
+user = ocfmail-ro
```

Comment:
good idea, but i'll wait on this for now (until future parser is ready probably)

afaik it's not possible to pass variables to regular ERB templates explicitly, it just reads from the current scope (and puppet doesn't let you redefine variables in a manifest), so it wouldn't be straightforward to parameterize the query (would need two separate manifests, or a "common" partial, which seems overkill)

with EPP it looks easier, but we don't have this yet:
https://docs.puppet.com/puppet/latest/reference/lang_template.html